packages/mongodb-rag-ingest/src/sources/mongodb-university/makeUniversityPages.ts

```diff
@@ -181,3 +202,26 @@ export function convertVideoTranscriptFromSrtToTxt(transcript: string): string {
     .join(" ")
     .trim();
 }
+
+/**
+  Helper function to create Markdown content for MongoDB University Learning Paths and Courses
+  based on titles, duration, and descriptions.
+ */
+export function generateMarkdown({
+  tiCatalogItem
+}: {
+  tiCatalogItem: TiCatalogItem;
+}): string {
+  const { name, description, nested_content } = tiCatalogItem;
+  const title = `# ${name}`;
+  const markdownContent = [title, description, `\n`];
+  if (nested_content) {
+    for (const nested of nested_content) {
+      const { name, duration, description, slug } = nested;
+      const title = `#### ${name}`;
```

Comment:
```suggestion
      const title = `## ${name}`;
```

should not go from H1 (`#`) --> to H4 (`####`) for nested sections. always go sequentially, H1-->H2-->H3 for nesting, etc.